content/docs/insights/concepts/_index.md

```diff
@@ -0,0 +1,107 @@
+---
+title_tag: "Pulumi Insights Overview"
+meta_desc: Pulumi Insights provides advanced search, analytics, and AI for your infrastructure as code.
+title: Pulumi Insights Concepts
+h1: Pulumi Insights Concepts
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+  insights:
+    parent: insights-home
+    identifier: insights-concepts
+    weight: 2
+aliases:
+- /docs/intro/insights/
+---
+
+Pulumi Insights provides advanced search, analytics, and AI for your infrastructure as code.
+
+It provides:
+
+- **Resource Search**: Explore all of your resources under management.
+  Filter, group, search, and sort resources by stack, project, or a number of other dimensions.
+
+- **Resource Search Aggregates**: See aggregates at a glance for your resources under management.
+  Start with a birds-eye view of your infrastructure, and leverage features like search, grouping, and filtering to dig deeper.
+
+- **Resource Search Favoriting and Sharing**: Save your updated view as a favorite in the left nav like stacks or share it with your teammates.
+
+- **Resource Search AI Assist**: (experimental) Use natural language processing to help craft search queries to explore your data.
+
+- **Data Export**: Export your Pulumi resources into your business intelligence tool of choice to go even further.
+
+- **Cloud Import**: Bring your own cloud provider account and import all your existing resources into Pulumi to see how things break down. Zero code required.
```

Comment:
this should be higher. it is the core of insights is to provide access to unmanaged resources

Reply:
done